### PR TITLE
Add tutorial styling

### PR DESCRIPTION
**Pull request by sendusyourbones**
Update styles on `/tutorials/new` page:
- Category chip styles updated to match styles on the view tutorial page
- Fixed bug with bolding the selected categories in the menu
- Set form width so that it doesn't span the full page width
- Made the overview field taller since we are asking for a few sentences
- Added some margins

Also added a few intro sentences explaining what the page is for and what happens next.

### CODE REVIEW

**Review by rileydoneil — COMMENTED** [review]
Looks great and works great! Are we planning on this page taking us to the add lesson page?

**Comment by sendusyourbones** [comment]
@rileydoneil Yep that functionality was just merged in in PR #127!

File: client/src/pages/AddTutorial.js [hunk 1]
```diff
@@ -295,7 +318,7 @@ export function AddTutorial() {
         <Button 
           type='submit' 
           variant='contained' 
-          sx={{ mt: 3, mb: 2 }}
+          sx={{ mt: 3, mb: 5 }}
         >
           Save Your Tutorial
```
Comment (sarah-jensen) [inline client/src/pages/AddTutorial.js:325]: Save button redirects to addLesson page, but I get a console error about reading properties of undefined for 'tutorial' on AddLessons.js

**Review by sarah-jensen — COMMENTED** [review]
Save button successfully adds info to db and redirects, but AddLesson throws an error upon loading

**Comment by sendusyourbones** [comment]
@sarah-jensen Should be fixed now, thanks!

**Review by sarah-jensen — APPROVED** [review]
looks good!

**Pull request merged**